tests/PowerPlaywright.IntegrationTests/ModelDrivenAppTests.cs

```diff
@@ -20,6 +20,21 @@ public async Task LoginAsync_ValidParameters_LogsInToApp()
         await this.Expect(appPage.Page).ToHaveURLAsync(MainPageRegex());
     }
 
+    /// <summary>
+    /// Tests that <see cref="ModelDrivenApp.ClientApi.OpenFormAsync"/> opens an entity form in create mode.
+    /// </summary>
+    /// <returns>A <see cref="Task"/> representing the asynchronous unit test.</returns>
+    [Test]
+    public async Task OpenFormAsync_EntityLogicalName_OpensEntityFormInCreateMode()
+    {
+        var appPage = await this.PowerPlaywright
+            .LaunchAppAsync(this.Context, Configuration.Url, TestAppUniqueName, this.User.Username, this.User.Password, this.User.TOTPSecret);
+
+        var newRecordPage = await appPage.ClientApi.OpenFormAsync("pp_record");
+
+        await this.Expect(newRecordPage.Form.Container).ToBeVisibleAsync();
```

Comment:
This assertion isn't bad, but I think we can improve it by asserting what we're really after here - the page has navigated to the correct URL. 

I think we can do something similar to what we've done for opening records from a grid (although there will need to be a slight tweak as there should be no `id` query string parameter).

`await this.Expect(recordPage.Page).ToHaveURLAsync(NewRecordFormUrlRegex()); `